### PR TITLE
Feature/microsite checklist module

### PR DESCRIPTION
**Pull request by rb1193**
As agreed with Will and Ying Zhou, this is a basic checklist without the group selection or accordion styles. Here's how it looks on desktop:

openlaw-checklist
___
NB - by making a pull request you confirm agreement to the [Contributors Agreement](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: src/components/Checklist.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <strong id={titleId} className="checklist-item-title">{item.title}</strong>
+        </header>
+        <ul id={containerId} className="checklist-sub-item-list">
+          {item.items.map((subItem, subItemIndex) => renderChecklistSubItem(subItem, subItemIndex, index))}
+        </ul>
+      </article>
+    )
+  }
+
+  const renderChecklistSubItem = (item, index, parentIndex) => {
```
Comment (danieljimeneznz) [inline src/components/Checklist.jsx:36]: Don't need curly bracket or return statement for arrow functions if you don't want
```suggestion
  const renderChecklistSubItem = (item, index, parentIndex) => (<li key={`checklist_item_${parentIndex}_subitem_${index}`} className="checklist-sub-item">
```

File: src/components/Checklist.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { useState } from "react"
+import produce from "immer"
+
+const CHECKLIST_CHECKED = true
+const CHECKLIST_EMPTY = false
+
+export default function Checklist({ id, items }) {
+  const [checklist, setChecklist] = useState(
+    items.map((item) => {
```
Comment (danieljimeneznz) [inline src/components/Checklist.jsx:10]: items is a list of items? naming seems a little bit confusing.

File: src/scss/Checklist.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+.checklist-item-checkbox img {
+    position: absolute;
+    transform: translate(-50%, -50%);
+}
+
+@media (max-width: 767px) {
+    .checklist-sub-item {
+        padding: 25px;
+    }
+}
```
Comment (danieljimeneznz) [inline src/scss/Checklist.scss:59]: How comes some files have no newlines at end of file but some do? Is there a prettier setting that could help make things more consistent?

Comment (rb1193) [reply src/scss/Checklist.scss:59]: I'm sure there is, one for @BenRauzi ?

File: src/components/Checklist.jsx (hunk 2 above)
Comment (rb1193) [reply src/components/Checklist.jsx:10]: Renamed this to subitems, which is a bit clearer I feel

File: src/components/Checklist.jsx (hunk 1 above)
Comment (rb1193) [reply src/components/Checklist.jsx]: Good call, changed locally

**Comment by rb1193** [comment]
@danieljimeneznz would you mind giving this a quick look over? If you're happy I'll give Ben a nudge to get it merged

**Pull request merged**